README.md

```diff
@@ -54,29 +54,44 @@ npm init @eslint/config@latest
 After that, you can run ESLint on any file or directory like this:
 
 ```shell
-./node_modules/.bin/eslint yourfile.js
+npx eslint yourfile.js
 ```
 
 ## Configuration
 
 After running `npm init @eslint/config`, you'll have an `eslint.config.js` (or `eslint.config.mjs`) file in your directory. In it, you'll see some rules configured like this:
 
 ```js
-import pluginJs from "@eslint/js";
-export default [ pluginJs.configs.recommended, ];
+export default [
+    {
+        files: ["**/*.js", "**/*.cjs", "**/*.mjs"],
+        rules: {
+            "prefer-const": "warn",
+            "no-constant-binary-expression": "error"
+        }
+    }
+];
```

Comment:
This would be confusing because this example is supposed to show a config file created by running  `npm init @eslint/config`, but there will never be any rule configurations in it. Perhaps we should leave it as is and then say "you can enable more rules..." and show an example with rules added to the config.

Reply:
Fair point. The problem with this as-is is that the following paragraph refers to rules that weren't even in the example.

Because this section is meant to teach about how to configure rules, it probably doesn't need to mention `npm init` at all.